saltstack/final/salt/cis-controls/scripts/cis_control.sh

```diff
@@ -53,14 +53,16 @@ git clone https://github.com/AutomateCompliance/AnsibleCompliancePlaybooks.git $
 ssh-keygen -A
 
 #Apply the SSG Ansible playbook
+mkdir /tmp/cis
+chmod 777 /tmp/cis
```

Comment:
Do we really need 777? Also I guess it could be applied recursively one time at the end of image burning

Reply:
I guess for the check itself not necessary but for the log collection yes. I do not think that it is a problem during burning time.